bgpd/bgp_snmp_bgp4v2.c

```diff
@@ -1412,6 +1445,83 @@ static struct variable bgpv2_variables[] = {
 	 {1, 9, 1, BGP4V2_NLRI_PATH_ATTR_UNKNOWN, 2, 16}},
 };
 
+int bgpv2TrapEstablished(struct peer *peer)
```

Comment:
Why even have a return value for this function?  And the following one?  I don't ever see anything but a return of 0?  What is the goal here?

Reply:
only coherency with previously existing function 
int bgp4TrapEstablished(struct peer *peer)